README.md

```diff
@@ -1,11 +1,61 @@
-# KEVM Ganache-CLI fork
+KEVM Ganache-CLI Fork
+=====================
+
+Installing KEVM
+---------------
 
-## Installing KEVM
 In order to run `kevm-ganache-cli`, you will need to install `kevm-vm`.
-We currently support [Ubuntu](#Ubuntu), [Arch](#Arch), and [MacOS](#MacOS).
+We currently support Ubuntu Bionic (18.04) and Arch Linux (MacOS coming soon).
+Here a specific GitHub release is used, feel free to replace the URL with a newer release from <https://github.com/kframework/evm-semantics/releases>.
+
+### Ubuntu Bionic (18.04) - fresh VM
+
+```sh
+sudo apt install nodejs npm
```

Comment:
Should we add `curl` here as well? My `lxc` container did not have it by default, for example.

Reply:
Done